dali/pipeline/util/copy_with_stride.cu

```diff
@@ -351,7 +351,7 @@ void CopyDlTensorBatchGpu(TensorList<GPUBackend> &output, std::vector<DLMTensorP
   }
   int element_size, ndim;
   ValidateBatch(element_size, ndim, dl_tensors, batch_size);
-  SmallVector<strided_copy::StridedCopyDesc, 128> sample_descs;
+  SmallVector<strided_copy::StridedCopyDesc, 32> sample_descs;
```

Comment:
Stack allocation was too large

Reply:
Well, it worked, so I wouldn't say with any amount of certainty that it was "too large". For our target it seems to have been OK.... Your call.

Reply:
Strided copy tests have had some random test failures in the CI for a while. Maybe there are still some issue there.

Reply:
If we ran out of stack it wouldn't be "some issues". It would be a hard fault (SIGSEGV? I don't think linux has a separate stack overflow error).